client/Components/Links/Links.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+import styles from "./Links.css";
+class Links extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  render() {
+    return (
+      <div id={styles.linksContainer}>
```

Comment:
Don't know if you have tried yet, but is it possible automate this part with a map? Seems repetitive, but you may have already tried. Just throwing it out there.